packages/extension-polkagate/src/util/workers/getNFTs.js

```
@@ -0,0 +1,163 @@
+    const items = await Promise.all(apis.map(({ api, chain, formattedAddresses }) =>
+      fetchNFTsForChain(api, formattedAddresses, chain)
+    ));
+
+    return items.flat();
+  } finally {
+    // Ensure all websocket connections are closed
+    apis.forEach(({ connections }) => closeWebsockets(connections));
+  }
+}
+
+onmessage = async (e) => {
```

Biome

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)